chart/templates/deployment.yaml

```diff
@@ -35,6 +35,14 @@ spec:
           securityContext:
             readOnlyRootFilesystem: true
             runAsNonRoot: true
+            runAsUser: 10001
+            runAsGroup: 10001
+            allowPrivilegeEscalation: false
+            capabilities:
+              drop:
+                - ALL
+            seccompProfile:
+              type: RuntimeDefault
```

Comment:
I also made these changes to the hosted chart: https://github.com/deliveryhero/helm-charts/pull/490